server/src/test/proto/spine/test/event/team.proto

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright 2017, TeamDev Ltd. All rights reserved.
```

Comment:
Copyright has changed. It's 2018 now. Please review the changeset on this matter.